site/docs/ru/guide/games.md

```diff
@@ -0,0 +1,101 @@
+# Игры
+
+## Введение
+
+Игры в Telegram - это очень интересная функция, с которой очень весело работать.
+Что вы можете сделать с ее помощью?
+Ответ: все, что угодно. Любую HTML5-игру, которую вы разработали, вы можете предоставить пользователям Telegram с помощью этой функции.
+(Да, это означает, что вам придется разработать настоящую игру на основе веб-сайта, которая находится в открытом доступе в интернете, прежде чем вы сможете интегрировать ее в своего Telegram-бота).
+
+## Настройка игр вашего бота через @BotFather
+
+Для простоты предположим, что к этому моменту вы уже настроили бота и игру, связанную с вашим ботом, написав [@BotFather](https://t.me/BotFather).
+Если вы еще не сделали этого, ознакомьтесь с этой [статьей](https://core.telegram.org/bots/games) от команды Telegram.
+
+> Примечание: мы будем изучать только разработку бота.
+> Разработка игры полностью зависит от разработчика.
+> Все, что нам нужно, это ссылка на HTML5-игру, размещенную в интернете.
+
+## Отправка игр через бота
+
+Мы можем отправить игру в grammY с помощью метода `replyWithGame`, который принимает в качестве аргумента название игры, созданной вами в BotFather.
+В качестве альтернативы можно использовать метод `api.sendGame` (grammY предоставляет все официальные методы [API бота](https://core.telegram.org/bots/api)).
+Преимущество использования метода `api.sendGame` в том, что вы можете указать `chat.id` конкретного пользователя для отправки.
+
+1. Отправка игры через `replyWithGame`
+
+   ```ts
+   // Мы будем использовать команду start для вызова метода ответа игрой
+   bot.command("start", async (ctx) => {
+     // Передайте имя игры, которую вы создали в BotFather, например "my_game".
+     await ctx.replyWithGame("my_game");
+   });
+   ```
+
+2. Sending a Game via `api.sendGame`
+
+   ```ts
+   bot.command("start", async (ctx) => {
+     // Вы можете получить идентификатор чата пользователя, которому нужно отправить игру, с помощью `ctx.from.id`.
+     // который дает вам идентификатор чата пользователя, вызвавшего команду start.
+     const chatId = ctx.from.id;
+     await ctx.api.sendGame(chatid, "my_game");
+   });
+   ```
+
+> [Помните](./basics#sending-messages) что вы можете указать дополнительные параметры при отправке сообщений, используя объект options типа `Other`.
+
+Вы также можете указать пользовательскую [встроенную клавиатуру](../plugins/keyboard#встроенная-клавиатура) для отображения кнопок в игре.
+По умолчанию будет отправлена кнопка с именем `Play my_game`, где _my_game_ - название вашей игры.
```

Comment:
Does the name of the button depend on the language in Telegram?

Reply:
Yes.

Reply:
This also means that the Russian translation should mention the label that can be found in Telegram when the app language is set to Russian.

Reply:
Interesting thing

![NVIDIA_Overlay_DBUyHH8wWu](https://github.com/grammyjs/website/assets/68379695/7776ea89-89f3-41f6-9164-9938921d526d)

Reply:
![image](https://github.com/grammyjs/website/assets/68379695/ed80f87b-b6ff-4d72-8b20-a93b16ad426c)

Reply:
Oof